pcdsdevices/dream_motion.py

```diff
@@ -97,3 +97,26 @@ class DREAM_GasNozzle(BaseInterface, GroupDevice):
     gas_nozzle_x = Cpt(BeckhoffAxis, ':MMS:X', kind='normal')
     gas_nozzle_y = Cpt(BeckhoffAxis, ':MMS:Y', kind='normal')
     gas_nozzle_z = Cpt(BeckhoffAxis, ':MMS:Z', kind='normal')
+
+
+class DREAM_SL3K4(BaseInterface, GroupDevice):
+    """
+    DREAM Motion Class
+    This class controls DREAM SL3K4 SmarAct based scatter slit
+
+    Parameters
+    ----------
+    prefix : str
+        DREAM:GSJN
```

Comment:
Maybe the documentation here is wrong
```
zlentz@tmo-control:~$ caget TMO:DREAM:MCS2:01:m7.DESC
TMO:DREAM:MCS2:01:m7.DESC      DREAM:SLIT:MMT:Y1
zlentz@tmo-control:~$ caget TMO:DREAM:MCS2:01:m12.DESC
TMO:DREAM:MCS2:01:m12.DESC     DREAM:SLIT:MMT:Y2
zlentz@tmo-control:~$ caget TMO:DREAM:MCS2:01:m9.DESC
TMO:DREAM:MCS2:01:m9.DESC      DREAM:SLIT:MMT:X2
zlentz@tmo-control:~$ caget TMO:DREAM:MCS2:01:m8.DESC
TMO:DREAM:MCS2:01:m8.DESC      DREAM:SLIT:MMT:X1
```
```suggestion
        TMO:DREAM:MCS2:01
```